templates/CRM/Admin/Form/Preferences/Display.tpl

```diff
@@ -209,6 +209,23 @@
       <td>&nbsp;</td>
       <td class="description">{ts}Sort name format for individual contact display names.{/ts}</td>
     </tr>
+
+    {if $config->userSystem->is_drupal EQ '1'}
```

Comment:
so this implies Drupal has no backend theme which might be true from a drupal head POV but not so much from a ' I have front end & back end pages POV'  - I don't think we need to solve that right now but some comments maybe.....

Reply:
Right, there's the *conceptual* notion of frontend/backend (which can apply on any technology), and then there's the technical notion (which has been clearly defined in WP/J but not as clearly on D/B).

I put the conditionals in there because `civicrm-core`+`civicrm-drupal` haven't formally had the distinction, and it seemed like it'd be confusing to display both `theme_frontend` and `theme_backend` to a Drupal admin (when `theme_frontend` would be non-functional).

Of course, less officially, there's `civicrmtheme.module` which does create this concept on Drupal. So it might make sense to align with that. Seamus has another comment+PR getting at this same point... will try to put some of the reply in there.

Reply:
@totten right - I was just thinking to get some code comments in there

Reply:
I agree that just a bit more in the documentation about how existing approaches to admin themes in Drupal could migrate or take advantage of this would be nice (but not strictly necessary at this point).